.github/workflows/constributors.yml

```
@@ -0,0 +1,18 @@
+name: CI
+on:
+    push:
+        branches:
+            - main
+
+permissions:
+  contents: write
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)